packages/components/src/components/stepper/stepper.stories.ts

```diff
@@ -0,0 +1,62 @@
+import { action } from '@storybook/addon-actions';
+
+export default{
+    title: 'Under Development/Stepper',
```

Comment:
Once you have done the changes, you can also remove the component from Under Development and move it to the general component section :)